packages/proof/src/generate-proof.ts

```diff
@@ -3,12 +3,12 @@ import type { Identity } from "@semaphore-protocol/identity"
 import { MAX_DEPTH, MIN_DEPTH } from "@semaphore-protocol/utils/constants"
 import { requireDefined, requireNumber, requireObject, requireTypes } from "@zk-kit/utils/error-handlers"
 import { packGroth16Proof } from "@zk-kit/utils/proof-packing"
+import { maybeGetSemaphoreSnarkArtifacts, type SnarkArtifacts } from "@zk-kit/utils"
```

Comment:
I didn't define an export for `./snark-artifacts` in https://github.com/privacy-scaling-explorations/zk-kit/pull/240
Should we do it?

Reply:
I have had trouble with that. As you can see there's no `crypto` either. The reason is that these modules have two versions: one for browsers and one for node. And I was not able to provide a conditional export for `crypto` with the right browser/node paths, because the Rollup conf to convert ts files to js only includes the node file. You could take a look and see if there's another solution.